text/0000-create-repo-issues.md

```diff
@@ -0,0 +1,101 @@
+# Meta
+[meta]: #meta
+- Name: Create Repo Issues
+- Start Date: 2020-08-06
+- Author(s): @dfreilich
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: [#04](https://github.com/buildpacks/rfcs/blob/main/text/0004-rfc-process.md) (which was [rfc#7](https://github.com/buildpacks/rfcs/pull/7))
+
+# Summary
+[summary]: #summary
+Once an RFC has been accepted, prior to it being merged in to the [RFC repo](http://github.com/buildpacks/rfcs), the author of the RFC should create issues in the appropriate Buildpacks repositories, linking them to the RFC pull request (using [Github references](https://docs.github.com/en/github/writing-on-github/autolinked-references-and-urls#issues-and-pull-requests))
+
+# Definitions
+[definitions]: #definitions
+
+* RFC &rarr; Request For Comments. It is the process followed by the Cloud Native Buildpacks project, as well as many other OSS ones, to provide a consistent and controlled path for new features to enter the spec, libraries, or tools, so that all stakeholders can be confident about the direction Cloud Native Buildpacks is evolving in.
+* FCP &rarr; Final Comment Period. This is step taken on a particular RFC when enough discussion of the tradeoffs have taken place and the team is in a position to make a decision. Before entering FCP, super majority of the team must sign off.
+* PR &rarr; (Github) Pull Request.
+
+# Motivation
+[motivation]: #motivation
+
+As the number of RFCs increase, it has become increasingly hard to ensure that a given RFC has been implemented, or will be implemented. This RFC proposes an amendment to the [RFC process](https://github.com/buildpacks/rfcs#rfc-process).
+
+The current RFC process [defines](https://github.com/buildpacks/rfcs#rfc-process) that:
+> the team member who merges the pull request should do the following ... Create a corresponding issue in the appropriate repo.
+
+In practice, that adds extra work for the team member who merges in the PR, and it may slip through the cracks occasionally. Additionally, the author of the RFC, as the one who proposed the change, is often the best person to create the issue, as the person who is most invested in that change. 
```

Comment:
I am not sure that I agree that "the person who proposed the change is the best person to create the issue". I fear that adding requirements like this makes the RFC process even more unapproachable for new contributors. I believe the best person to create the issue is a maintainer of the relevant repo. 

As an implementation maintainer I currently try to create spec and lifecycle issues as needed when RFCs merge (examples: https://github.com/buildpacks/spec/issues/106, https://github.com/buildpacks/lifecycle/issues/354).

This makes sense to me b/c I have the lifecycle knowledge and permissions required to subdivide, label, organize in milestones etc. Example: RFC-0053 affected two APIs therefore needs two separate spec issues ( https://github.com/buildpacks/spec/issues/134, https://github.com/buildpacks/spec/issues/133) I don't think think every RFC author should have to think about that level of detail (although stephen wrote this one and he certainly could have ;p).

On the other hand I _do_ think every subteam maintainer should be thinking about what each RFC means for their subteam as RFCs go through and should be planning/organizing accordingly.

Reply:
This mirrors my comment above. I'm very much in favor of issues being created from RFCs, but I think this should be handled by the subteams (not just the maintainers). Would you be open to that change @dfreilich?

Reply:
I'd definitely be open to that, that makes sense. One suggestion - do people think it makes sense to add additional label(s), like `needs-issues-impl`, `needs-issues-platform`, etc? In that context, when an RFC is put in FCP, the appropriate labels could be added, and they'd need to be removed for an RFC to be merged in. Or would that be over-engineering the solution...